package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This package exists only to hold eslint and It's dependency",
   "main": "n.a",
   "scripts": {
-    "dev": "sed -i 's/DEV_SERVER.*/DEV_SERVER = 1/' server/.env.atlas; concurrently \"cd client && npm start\" \"cd server && npm run atlas\"",
+    "dev": "sed -i 's/DEV_SERVER.*/DEV_SERVER = 1/' server/.env.atlas; concurrently -s --kill-others --names \"BUILD,SERVER\" -c \"bgBlue,bgGreen\" \"cd client && npm start\" \"cd server && npm run atlas\"",
```

Comment:
This line is a but brutal. & what does it mean?

Reply:
sed: edit .env.atlas config file, and set DEV_SERVER value to 1.
concurrently: start 2 parallel process in the terminal
   - frontend dev server, logs with a blue [ BUILD ] tag,
   - backend server, logs with a green [ SERVER ] tag.

Reply:
thanks :)